me.brokenby.PdfMetadataEditor.yml

```diff
@@ -0,0 +1,38 @@
+app-id: me.brokenby.PdfMetadataEditor
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+commands:
+  - '/app/bin/pmedit-cli'
+modules:
+  - name: pdf-metadata-editor-app
+    buildsystem: simple
+    build-commands:
+      - mv -v * ${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://github.com/zaro/pdf-metadata-editor/releases/download/v3.0.0/pdf-metadata-editor-3.0.0-linux-portable.tar.gz
```

Comment:
Is this x86_64 only? Why not build from source? Is aarch64 available?

If you want x86_64 only you need a flathub.json https://docs.flathub.org/docs/for-app-authors/maintanance#only-build-on-x86_64

Reply:
You have to manually add all maven sources by hand, there's a few apps doing it I think but maven dependencies can be a bit of hit or miss because some of them assume a network connection is present at build time which isn't true on Flathub.